lib/install/config/loaders/core/assets.js

```diff
@@ -1,12 +1,30 @@
-const { env, publicPath } = require('../configuration.js')
+const Url = require('url-parse')
+const { env, paths, publicPath } = require('../configuration.js')
+
+// Make sure host has slashes
+const ensureSlashes = ({ href, slashes }) => {
+  if (slashes) { return `${href}/${paths.entry}/` }
+  return `//${href}/${paths.entry}/`
```

Comment:
Yepp, done the same when you suggested the other day but it seems eslint doesn't like it. If an `if` block contains a return statement, the else block becomes unnecessary. (one of those JS things 😉 )